test/e2e/tests/remote-feature-flag/remote-feature-flag.spec.ts

```diff
@@ -45,4 +51,33 @@ describe('Remote feature flag', function (this: Suite) {
       },
     );
   });
+
+  it('offers the option to pass into manifest file for developers', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder()
+          .withMetaMetricsController({
+            metaMetricsId: MOCK_META_METRICS_ID,
+            participateInMetaMetrics: true,
+          })
+          .build(),
+        manifestFlags: {
```

Comment:
4. Usage in e2e test: override remoteFeatureFlags and assert the result to be `MOCK_CUSTOMIZED_REMOTE_FEATURE_FLAGS`